backends/platform/androidsdl/androidsdl-sdl.cpp

```diff
@@ -42,7 +44,101 @@ void OSystem_ANDROIDSDL::initBackend() {
 
 	if (!ConfMan.hasKey("gfx_mode"))
 		ConfMan.set("gfx_mode", "2x");
+	
+	if (!ConfMan.hasKey("swap_menu_and_back"))
```

Comment:
it's only short expression of 'swap menu and back buttons'
A lot of new device don't have physical menu button and for call menu i used back button.
But i received some feedback where users ask about possiblity to use menu (if it present on device) as menu and back as skip. That was implemented. Now it possible to switch it via main 'Options' menu.